spec/services/worldpay/refund_service_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Worldpay
+  RSpec.describe RefundService do
+    let(:payment) { double(:payment) }
+    let(:result) { described_class.run(payment: payment) }
+
+    describe ".run" do
+      context "when the payment is not a worldpay payment nor a worldpay_missed payment" do
+        it "returns nil" do
```

Comment:
When I looked at the service, assuming it gets past these checks we return `true` or `false` based on whether `parsed_reponse.xpath("//paymentService/reply/ok/refundReceived")` is `present?`.

I feel like we should always be returning either `true` or `false` because of this, rather than `true`, `false` or `nil`.

If we need 3 return states, then possibly we could be more explicit, but I think from the code I've seen here we are treating `nil` as false so always returning `false` in this scenario ("when the payment is not a worldpay payment nor a worldpay_missed payment") wouldn't break anything.

Reply:
Yep make sense :)